extensions/udapdr/scripts/colbert/distillation/ranking_scorer.py

```diff
@@ -0,0 +1,52 @@
+import tqdm
```

Comment:
Identical to `./primeqa/extensions/ir_benchmarking/colbert/distillation/ranking_scorer.py`
Functionally identical (different imports) as `./primeqa/primeqa/ir/dense/colbert_top/colbert/distillation/ranking_scorer.py`